src/components/FAQ/FAQ.js

```diff
@@ -1,61 +1,60 @@
 import React from 'react';
-import './FAQ.scss';
+import './FAQ.css';
 
 function FAQ() {
   function toggleState(id) {
-    if (document.getElementById(id).style.visibility === '') {
-      document.getElementById(id).style.visibility = 'visible';
-      document.getElementById(id).style.opacity = 1;
+    if (document.getElementById(id).style.display === '') {
+      document.getElementById(id).style.display = 'block';
     }
 
-    if (document.getElementById(id).style.visibility === 'hidden') {
-      document.getElementById(id).style.visibility = 'visible';
-      document.getElementById(id).style.opacity = 1;
-    } else if (document.getElementById(id).style.visibility === 'visible') {
-      document.getElementById(id).style.visibility = 'hidden';
-      document.getElementById(id).style.opacity = 0;
+    if (document.getElementById(id).style.display === 'none') {
+      document.getElementById(id).style.display = 'block';
+    } else if (document.getElementById(id).style.display === 'block') {
+      document.getElementById(id).style.display = 'none';
     }
   };
 
   return (
-    <div class="faq-section" id="faq">
-      <div class="row" style={{minHeight: "515px"}}>
+  <div>
+    <div class="page-title">
+      <div class="letter">F</div>
+      <div class="letter">A</div>
+      <div class="letter">Q</div>
+    </div>
+    <div class="faq-section">
+      <div class="faq-row" style={{minHeight: "515px"}}>
         <div class="module">
-          <div class="title" onClick={() => {toggleState("one")}}>What is a hackathon?</div>
-          <div class="answer" id="one">A hackathon is a place where you can collaborate with others and express your creativity to create a product that solves some real-world problem. At GraceHacks, students of all different backgrounds and experiences will gather together to develop an idea and realize that solution all within the span of 12 hours!</div>
+          <div class="faq-title" onClick={() => {toggleState("one")}}><div class= 'ques'>What is a hackathon?</div> <img src={require("./vector.png")} class="arrow"/></div>
```

Comment:
Make sure to keep class styling consistent and remove the space in `class= 'ques'`.